pulp/tests/test_pulp.py

```diff
@@ -588,12 +590,7 @@ def test_pulp_075(self):
             x = LpVariable("x", 0, 4, const.LpContinuous, obj + b)
             y = LpVariable("y", -1, 1, const.LpContinuous, 4 * obj - c)
             z = LpVariable("z", 0, None, const.LpContinuous, 9 * obj + b + c)
-            if self.solver.__class__ in [
-                CPLEX_CMD,
-                COINMP_DLL,
-                YAPOSIB,
-                PYGLPK,
-            ]:
+            if self.solver.__class__ in [CPLEX_CMD, COINMP_DLL, YAPOSIB, PYGLPK]:
```

Comment:
Most changes here were automatically made by black. I only added HiGHS to this file